cellpack/autopack/__init__.py

```diff
@@ -177,8 +177,8 @@ def checkPath():
     "autoPACKserver": autoPACKserver,
     "autopackdir": autopackdir,
     "autopackdata": appdata,
-    f"{DATABASE_IDS.GITHUB}:": autoPACKserver,
-    f"{DATABASE_IDS.FIREBASE}:": None,
+    DATABASE_IDS.GITHUB + ":": autoPACKserver,
+    DATABASE_IDS.FIREBASE + ":": None,

```

Comment:
I'm imagining this is somehow related to the python version change, but the string formatting wasn't working correctly and these keys were just getting set to "DATABASE_IDS.GITHUB:" and "DATABASE_IDS.FIREBASE:", which was causing errors. This change makes it so the keys are "github:" and "firebase:", as expected